services/hotelsService.js

```diff
@@ -40,19 +40,18 @@ exports.getFreeRooms = async (req) => {
 }
 
 exports.getHotels = async (req) => {
-  if (req.query && req.query.pageNumber && req.query.pageSize) {
-    const { pageNumber, pageSize } = req.query
+  const hotelsLength = await Hotel.countDocuments()
 
-    const hotelsLength = await Hotel.count()
-    const hotels = await Hotel.find()
-      .skip((+pageNumber - 1) * +pageSize)
-      .limit(+pageSize)
+  const pageNumber = req.query.pageNumber ? req.query.pageNumber : 1
+  const pageSize = req.query.pageSize ? req.query.pageSize : hotelsLength
```

Comment:
I think we should never return all hotels, if pageSize is not given, then we should return some default pageSize, e.g. 50 or so.